api/src/pcapi/core/finance/factories.py

```diff
@@ -224,12 +224,12 @@ def _create(
         *args: typing.Any,
         **kwargs: typing.Any,
     ) -> models.CustomReimbursementRule:
-        if "rate" in kwargs:
+        if kwargs.get("rate"):
```

Comment:
Détail : Attention la condition ne donne plus le même résultat si `rate` vaut `0` !
C'est certes un cas peu probable...
```suggestion
        if kwargs.get("rate") is not None:
```